cmd/nomos/version/version.go

```diff
@@ -136,44 +140,61 @@ func filterConfigs(contexts []string, all map[string]*rest.Config) map[string]*r
 	return cfgs
 }
 
-func lookupVersionAndMode(ctx context.Context, cfg *rest.Config) (string, *bool, error) {
+func getImageVersion(deployment *v1.Deployment) (string, error) {
+	reconcilerImage := strings.Split(deployment.Spec.Template.Spec.Containers[0].Image, ":")
```

Comment:
Suggest checking if the container's name is the `reconciler-manager`. The reconciler-manager has two containers, using the container index is not reliable.

Reply:
updated to search for container named reconciler-manager